package.json

```diff
@@ -4,11 +4,54 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "fix:other": "yarn prettier --write",
+    "fix:js": "yarn test:js --fix",
+    "fix": "yarn fix:js && yarn fix:other",
+    "prettier": "prettier \"**/*.{json,md,scss,yaml,yml}\"",
+    "test:other": "yarn prettier --list-different",
+    "test:js": "eslint --ignore-path .gitignore --ignore-path \"**/*.{js,jsx}\"",
+    "test": "yarn test:other && yarn test:js",
+    "build": "babel src/ -d /",
+    "prepublish": "yarn run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/react-native-community/react-native-hooks"
+  },
+  "bugs": {
+    "url": "https://github.com/react-native-community/react-native-hooks/issues"
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.{json,md,scss,yaml,yml}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "precommit": "flow check && lint-staged"
+    }
   },
   "author": "Nader Dabit",
   "license": "ISC",
   "dependencies": {
+    "flow": "^0.2.3",
+    "flow-bin": "^0.92.1",
+    "husky": "1.3.1",
```

Comment:
Why husky, flow, lint and others added in dependencies ?

I think you can add in devDependencies

Reply:
yup, they should be in dev dependencies but they were not working for some reason so i kept them in dependencies, i will migrate them to devDependencies in the next commit if it worked.